datasetinsights/estimators/faster_rcnn.py

```diff
@@ -22,6 +22,594 @@
 MAX_BOXES_PER_IMAGE = 100
 logger = logging.getLogger(__name__)
 DEFAULT_ACCUMULATION_STEPS = 1
+TRAIN = "train"
+VAL = "val"
+TEST = "test"
+
+
+class FasterRCNN(Estimator):
+    """
+    https://github.com/pytorch/vision/tree/master/references/detection
+    https://arxiv.org/abs/1506.01497
+    Args:
+        config (CfgNode): estimator config
+        writer: Tensorboard writer object
+        checkpointer: Model checkpointer callback to save models
+        device: model training on device (cpu|cuda)
+        local_rank (int): (optional) rank of process executing code
+        gpu (int): (optional) gpu id on which code will execute
+    Attributes:
```

Comment:
Attributes doc mismatch with public instance variables for this class. Can you improve it?